tests/integration/event_filters_test.go

```diff
@@ -2377,6 +2377,9 @@ func Test_EventFilters(t *testing.T) {
 }
 
 const (
+	expectFromSystem    = ""
```

Comment:
Not exactly sure what this commit does, what info it provides that could be useful, could you elaborate in the commit message and here?